flink-end-to-end-tests/flink-end-to-end-tests-aws-kinesis-firehose/pom.xml

```diff
@@ -51,6 +51,14 @@
 			<scope>test</scope>
 			<type>test-jar</type>
 		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-aws-base</artifactId>
```

Comment:
its a bit weird that this is now (supposedly) required. Irrespective of the changes made, aws-base wasn't on the classpath in the e2e test because so far it only used test-jars which don't bring in transitive dependencies.